sdk/android/sdk/src/main/java/com/ooyala/android/ooyalaskinsdk/util/ReactUtil.java

```diff
@@ -21,6 +21,7 @@ public static void setJSServer(String host, Context context) {
         SharedPreferences mPreferences = PreferenceManager.getDefaultSharedPreferences(context);
         SharedPreferences.Editor editor = mPreferences.edit();
         editor.putString("debug_http_host", host);
+        editor.putBoolean("js_dev_mode_debug", false);
```

Comment:
what does this do?  does this hide errors? We don't want to hide errors at them oment